packages/vscode-extension/src/workspace.ts

```diff
@@ -168,7 +168,7 @@ export class Workspace {
 			binPath += '.exe';
 		}
 
-		if (!path.isAbsolute(binPath)) {
+		if (!path.isAbsolute(binPath) && !binPath.startsWith('~')) {
```

Comment:
This works for this block, but I don't think it will work for the for lookup below.

Maybe do an "if starts with ~, replace with home dir" approach.